libs/ui/src/templates/JoinTemplate/JoinTemplate.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Story, Meta } from '@storybook/react'
+
+import { JoinTemplate, JoinTemplateProps } from '.'
```

Comment:
I'm not sure if it causes circular dependency error but I would import them from their own files like following:

```suggestion
import { JoinTemplate } from './JoinTemplate'
import { JoinTemplateProps } from './types'
```

Reply:
And if you don't have multiple component files in the folder, you don't need to create `types.ts`, you can put the type in the component file.